### DISCUSSION TITLE
The README example needs to be updated

### DISCUSSION THREAD

**Discussion by Nickiel12** [hecrj/iced#1250] [General]
Hello!
I am trying Iced for the first time today, and I got really confused because of the readme. I followed the "Overview" section, but I was getting errors, so I checked with the example for it, and it was different in a few critical places. 
Most important, is the fact that, nowhere in the overview, does it mention that `impl Counter` is actually `impl Sandbox for Counter`, which is kinda important. There are other things, like `fn view(...) -> Column<Message> {` which is actually `Element<Message>` in the examples, and that there needs to be a call to `Column::new()..... .into()` at the end of the view function.
These things threw me off because the only starting point I know is the readme, and it had incorrect information.
Something else that I haven't seen any explanation about, is how does the `Text` widget know to update?

**Comment by Nickiel12** [comment]
also, the 0.3 docs for `Sandbox` has links to 0.2 examples

**Comment by Nickiel12** [comment]
It would also be nice to have links to where to find more information about different layouts. Assuming from the fact that we use a `Column` type, are there others?

**Reply by Nickiel12** [reply]
Maybe recommend at the end of the readme to run the `tour` package example for an example of the basic widgets available.

**Comment by hecrj** [comment]
Duplicate of #531.

The **overview** is a brief introduction to the Elm Architecture. It focuses on the library concepts. It's not meant to be a detailed example. This should be obvious since there is no `main` function!

The code is correct and not outdated! You don't need to always return  an `Element`.

Using `Sandbox` is only one way of many to use the library. It is not essential in any way (e.g. if you write your own event loop).

There already links to the documentation and the examples at the end of the overview!

That said, I will try to further clarify that the code needs to be plugged to `iced` before running it.
